src/main.py

```diff
@@ -0,0 +1,47 @@
+import torch
+from torch.utils import data
+from torchvision import transforms
+from dataloader import MyDataset
+from dataloader import UnZipFolder
+import shutil
+import os
+
+# UnZip DataSet
```

Comment:
The same. I think the data has to be already unzipped/ready when starting the main.
I would remove that logic from main.

Reply:
Done